resources/schema.json

```diff
@@ -42,6 +42,11 @@
         }
       ]
     },
+    "transitionAnimation": {
```

Comment:
We try to keep all options in the `options` object. Doing so will show the options next to the preview in the Q Editor. Have a look [at this option](https://github.com/nzzdev/Q-imageslider/blob/dev/resources/schema.json#L152) in the imageslider tool as a reference. Additionally, I would call the option similar to the option in the imageslider tool "Überblendung aktivieren". So the user knows that activating this option will activate the animation.